### PR TITLE
show transfer error regardless of model type; disable transfer option…

### PR DESCRIPTION
**Pull request by nmorasb**
… for composite measures

## JIRA Ticket
[MAT-5682](https://jira.cms.gov/browse/MAT-5682)

## Checklist

- [ ] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [ ] Tests are included and test edge cases.
- [ ] Tests have been run locally and pass.

## Screenshots (if appropriate)

- [ ] None applicable

### CODE REVIEW

File: src/main/java/mat/client/export/ManageExportPresenter.java [hunk 1]
```diff
@@ -53,7 +53,7 @@ private void initializeExportView() {
 		exportView = new ManageMeasureExportView();
 		new ManageMeasureExportPresenter(exportView, result, manageMeasurePresenter);
 		exportView.setExportOptionsBasedOnVersion(result.getHqmfReleaseVersion(), result.getIsComposite(), result.getMeasureModel(), isTransferableToMadie);
-		if ("FHIR".equals(result.getMeasureModel()) && !isTransferableToMadie) {
+		if (!isTransferableToMadie || result.getIsComposite()) {
 			exportView.displayErrorMessage("This measure cannot be transferred to MADiE because either you are not the owner or a version of this measure has already been transferred.");
```
Comment (jkotanchik-SB) [inline src/main/java/mat/client/export/ManageExportPresenter.java:57]: Optional: Add language to the error for composite measures.

**Review by jkotanchik-SB — APPROVED** [review]
